<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleLogo</class>
 <widget class="QWidget" name="VehicleLogo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(51,153,255);
}
QLabel
{
	border:1px solid black;
	font:20px;
}</string>
  </property>
  <widget class="QLabel" name="labelM1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>35</y>
     <width>130</width>
     <height>65</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>M1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelM2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>35</y>
     <width>130</width>
     <height>65</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>M2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelC1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>35</y>
     <width>120</width>
     <height>65</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>C1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelC2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>35</y>
     <width>120</width>
     <height>65</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>C2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelNp">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>35</y>
     <width>100</width>
     <height>65</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>NP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelLeftHead">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>35</y>
     <width>65</width>
     <height>65</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);
border-radius:30px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelRightHead">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>35</y>
     <width>101</width>
     <height>65</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);
border-radius:30px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPan">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/LowPosition.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelLfitDirection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>41</width>
     <height>65</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelRightDirection">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>35</y>
     <width>41</width>
     <height>65</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurrentStation">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelNextStation">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>0</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelDoor9">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>35</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>36</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
background-color: rgb(255, 255, 127);
font:15px;</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDoor10">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>79</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label10">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>79</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
background-color: rgb(255, 255, 127);
font:15px;</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDoor1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>35</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>79</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
background-color: rgb(255, 255, 127);
font:15px;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>36</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
background-color: rgb(255, 255, 127);
font:15px;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDoor2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>79</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelDoor6">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>79</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>464</x>
     <y>36</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
background-color: rgb(255, 255, 127);
font:15px;</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDoor5">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>35</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label6">
   <property name="geometry">
    <rect>
     <x>464</x>
     <y>79</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
background-color: rgb(255, 255, 127);
font:15px;</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDoor4">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>79</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelDoor3">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>35</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>36</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
background-color: rgb(255, 255, 127);
font:15px;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>79</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
background-color: rgb(255, 255, 127);
font:15px;</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label8">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>79</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
background-color: rgb(255, 255, 127);
font:15px;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDoor7">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>35</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelDoor8">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>79</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label7">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>36</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
background-color: rgb(255, 255, 127);
font:15px;</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label12">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>79</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
background-color: rgb(255, 255, 127);
font:15px;</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDoor11">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>35</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelDoor12">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>79</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label11">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>36</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
background-color: rgb(255, 255, 127);
font:15px;</string>
   </property>
   <property name="text">
    <string>11</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>labelRightHead</zorder>
  <zorder>labelLeftHead</zorder>
  <zorder>labelM1</zorder>
  <zorder>labelM2</zorder>
  <zorder>labelC1</zorder>
  <zorder>labelC2</zorder>
  <zorder>labelNp</zorder>
  <zorder>labelPan</zorder>
  <zorder>labelLfitDirection</zorder>
  <zorder>labelRightDirection</zorder>
  <zorder>labelCurrentStation</zorder>
  <zorder>labelNextStation</zorder>
  <zorder>labelDoor9</zorder>
  <zorder>label9</zorder>
  <zorder>labelDoor10</zorder>
  <zorder>label10</zorder>
  <zorder>labelDoor1</zorder>
  <zorder>label2</zorder>
  <zorder>label1</zorder>
  <zorder>labelDoor2</zorder>
  <zorder>labelDoor6</zorder>
  <zorder>label5</zorder>
  <zorder>labelDoor5</zorder>
  <zorder>label6</zorder>
  <zorder>labelDoor4</zorder>
  <zorder>labelDoor3</zorder>
  <zorder>label3</zorder>
  <zorder>label4</zorder>
  <zorder>label8</zorder>
  <zorder>labelDoor7</zorder>
  <zorder>labelDoor8</zorder>
  <zorder>label7</zorder>
  <zorder>label12</zorder>
  <zorder>labelDoor11</zorder>
  <zorder>labelDoor12</zorder>
  <zorder>label11</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
